app/settings/roles/editor.directive.js

```diff
@@ -18,6 +18,7 @@ function (
             $scope.save = $translate.instant('app.save');
             $scope.saving = $translate.instant('app.saving');
             $scope.saving_role = false;
+            // @uirouter-refactor this is very wrong.
```

Comment:
self note because I noticed this old code from last week that is 100% wrong. We are not guaranteed a $resolve everywhere so we should get it from $state if possible